src/test/unit/fetch-assembly.test.js

```diff
@@ -0,0 +1,241 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import { fetchAssembly } from 'firefox-profiler/utils/fetch-assembly';
+import { ensureExists } from '../../utils/flow';
+import type { NativeSymbolInfo, Lib } from 'firefox-profiler/types';
+
+describe('fetchSource', function () {
```

Comment:
I guess this should be fetchAssembly and not fetchSource.

Also, thanks for the tests!

Reply:
Good catch, fixed.